### PR TITLE
Make the npm package support ethpm package.

### PR DESCRIPTION
**Pull request by leviadam**

### CODE REVIEW

File: ethpm.json [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    "Adam Levi <adam@daostack.io>",
+    "Jelle Gerbrandy <jelle@gerbrandy.com>"
+  ],
+  "keywords": [
+    "solidity",
+    "ethereum",
+    "smart",
+    "contracts",
+    "security",
+    "daostacl",
```
Comment (orenyodfat) [inline ethpm.json]: daostack

Comment (dkent600) [reply ethpm.json]: daostacl?

File: package.json [hunk 2]
```diff
@@ -65,12 +52,15 @@
     "backfeed",
```
Comment (orenyodfat) [inline package.json]: remove backed

**Comment by dkent600** [comment]
Please this this comment:  https://github.com/daostack/arc-js/pull/2#issuecomment-354114767

**Pull request merged**